### PR TITLE
feat(commands): add comment to get hrbot version from package.json

### PR DESCRIPTION
**Pull request by PatrickDohn**
Add command to get latest HrBot version.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -31,6 +32,7 @@ app.event('app_mention', async ({ event, say }) => {
   const commandsList = _.keys(COMMANDS);
   const matches = stringSimilarity.findBestMatch(text, commandsList);
   const { bestMatch } = matches;
+  const pjson = require('../package.json');
```
Comment (davidholyko) [inline src/app.js:35]:
```
const { version } = require('../package.json');
````

File: src/commands/commands.js [hunk 2]
```diff
@@ -21,6 +22,7 @@ const COMMANDS = {
   'get me our holidays': TOPICS.GET_HOLIDAYS,
   'can I see the spreadsheet': TOPICS.GET_SPREADSHEET,
   'help please': TOPICS.GET_HELP,
+  'get version': TOPICS.GET_VERSION,
```
Comment (davidholyko) [inline src/commands/commands.js:25]: add more commands

```
'what version is this bot'
```

Comment (PatrickDohn) [reply src/commands/commands.js]: Taken care of!

File: src/app.js (hunk 1 above)
Comment (PatrickDohn) [reply src/app.js]: Taken care of!

File: src/app.js [hunk 3]
```diff
@@ -31,6 +32,7 @@ app.event('app_mention', async ({ event, say }) => {
   const commandsList = _.keys(COMMANDS);
   const matches = stringSimilarity.findBestMatch(text, commandsList);
   const { bestMatch } = matches;
+  const { version } = require('../package.json');
```
Comment (davidholyko) [inline src/app.js:35]: this require line should be at the very top of the file

Comment (PatrickDohn) [reply src/app.js]: Fixed!

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**